README.md

```diff
@@ -143,6 +143,191 @@ print(result[1].token_str)  # technology
 ## Text Classification 
 
 
+Initialize a HappyTextClassification() object to perform text classification. 
```

Comment:
`HappyTextClassification()` is a constructor; not a type

Reply:
I changed it to "HappyTextClassification" in d3b4534